examples/kfp-pipelines/superworkflows/ray/kfp_v2/superpipeline_noop_docId_ededup_v2_wf.py

```diff
@@ -16,27 +16,53 @@
 import kfp.components as comp
 import kfp.dsl as dsl
 from universal.doc_id.kfp_ray.doc_id_wf import doc_id
+from universal.ededup.kfp_ray.ededup_wf import ededup
 from universal.noop.kfp_ray.noop_wf import noop
 
 from kfp import dsl
 
 
 noop_image = "quay.io/dataprep1/data-prep-kit/noop-ray:latest"
 doc_id_image = "quay.io/dataprep1/data-prep-kit/doc_id-ray:latest"
+ededup_image = "quay.io/dataprep1/data-prep-kit/ededup-ray:latest"
 
+# A list of the transform names, arranged in the order of their execution within the superpipeline.
+ordered_transforms = ["noop", "doc_id", "ededup"]
 
-def _remove_unused_params(d: dict[str, Any]) -> None:
+def _remove_unused_params(d: dict[str, Any], remove_params: list = None) -> None:
```

Comment:
do we really need this function, I've started to think about the separation of the input parameters use an additional prefix. Let's talk